README.md

```diff
@@ -25,15 +25,16 @@ npm install --save komada
 Create a file called `app.js` (or whatever you prefer) which will initiate and configure Komada.
 
 ```js
-const komada = require('komada');
+const komada = require("komada");
+
 komada.start({
-  "botToken": "your-bot-token",
-  "ownerID" : "your-user-id",
-  "clientID": "the-invite-app-id",
-  "prefix": "+",
-  "clientOptions": {
-    "fetchAllMembers": true
-  }
+  botToken: "your-bot-token",
+  ownerID: "your-user-id",
+  clientID: "the-invite-app-id",
+  prefix: "+",
+  clientOptions: {
+    fetchAllMembers: true,
```

Comment:
Leave it blank and have a comment next to it or smth saying refer to discord.js documentation